dts/arm/nxp/nxp_kw41z.dtsi

```diff
@@ -19,7 +19,6 @@
 		pwm-0 = &pwm0;
 		pwm-1 = &pwm1;
 		pwm-2 = &pwm2;
-		rtc-0 = &rtc0;
```

Comment:
Why are we removing the RTC aliases?

Reply:
because its not used for anything.  The goal is to transition away from aliases.  We might keep the around for use in samples, but but beyond that drivers shouldn't be utilizing them.